Sources/SwiftGraphQl/Lexer.swift

```diff
@@ -150,8 +150,10 @@ struct GraphQlLexer {
   static let assignment = "=" ^^ { _ in StreamToken.assignment }
   static let exclamation = "!" ^^ { _ in StreamToken.exclamation }
 
-  static let nameStart = match(CharacterSet.alphanumerics)
```

Comment:
this was incorrect anyway: identifiers can't begin with numbers.

Reply:
Yep. Now we're fully spec compliant 🎉